components/Footer.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <StyledFooter>
+      <Link href="/"> 🏋️‍♀️ </Link>
+    </StyledFooter>
+  );
+}
+
+const StyledFooter = styled.footer`
+  width: 100%;
+  height: 80px;
+  position: fixed;
+  bottom: 0;
+  display: flex;
+  text-align: center;
+  background: lightgrey;
+  flex-shrink: 0;
+  justify-content: center;
+  align-items: center;
+  border-radius: 10px 10px 0px 0px;
+
+  a {
+    font-size: 2rem;
```

Comment:
yes & no :D 
According to the current step, I could remove the Link and replace it with a p tag, so I can add the Emoji and then style it (in the future it will be an icon, as a Link for the page). 

But according to the Link import from next, the style option for the Link component is actually the a tag :) 

Would you consider, changing it up to the p tag at first? 'Cause I'm not quite sure.